ui/components/connections/index.js

```diff
@@ -264,14 +297,8 @@ function Connections({
         },
         customBodyRender : (value, tableMeta) => {
           return (
-            <Tooltip
-              title={connections ? connections[tableMeta.rowIndex]?.metadata?.server_location : ""}
-              placement="top"
-            >
-              <Link
-                href={connections ? connections[tableMeta.rowIndex]?.metadata?.server_location : ""}
-                target="_blank"
-              >
+            <Tooltip title={tableMeta.rowData[1]} placement="top" >
```

Comment:
Can we move away from this type of indexing? Since the data is structured we know what attribute is on which index, this avoids future confusion.